Command/SystemInfoDumpCommand.php

```diff
@@ -8,25 +8,55 @@
  */
 namespace EzSystems\EzSupportToolsBundle\Command;
 
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
```

Comment:
Glad we've cleared it up.

The notes above about consistency in services & classes names must be taken care of anyway I think. We are indeed close from a legacy domain concept, and we must make sure there are no inconsistencies there.